### PR TITLE
Color palette content

### PR DESCRIPTION
**Pull request by sarojgurung**

### CODE REVIEW

File: docs/guides/foundations/colours.md [hunk 1]
```diff
@@ -5,11 +5,58 @@ custom_edit_url: null
...
+These color are part of the secondary palette and can be used within in components other than infographics.
+
+![Secondary data visualization](../assets/color-secondary-data-vis.svg)
+
+The following colors, also termed as fuel color in GEL and are restricted to data visualization and infographics.
+
+![Fuel data visualization](../assets/color-fuel-data-vis.svg)
+
+
+## Resource
```
Comment (danielwang) [inline docs/guides/foundations/colours.md:60]: please add reference at the bottom too

```
## See also
 - [Design colour tokens](design-tokens/color)
 - [AEMO Branding colour](../../branding/colours/colour-palette)
```

Comment (danielwang) [reply docs/guides/foundations/colours.md:60]: make sure the links works

**Comment by sarojgurung** [comment]
@danielwang added those two resources under see more

**Review by danielwang — APPROVED** [review]
LGTM

**Pull request merged**